src/utils/kit.ts

```diff
@@ -1,4 +1,6 @@
-import { importModule, tryResolveModule } from './esm'
+import { tryResolveModule } from '@nuxt/kit'
```

Comment:
We use `tryResolveModule` to resolve kit 😅 I prefer to keep a standalone version of resolver in cli to avoid behavior conflicting.